techniques/html/H79.html

```diff
@@ -12,62 +11,60 @@
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>All technologies that contain links.</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/link-purpose-in-context">Success Criterion 2.4.4: Link Purpose (In Context)</a> (Sufficient as a way to meet an unwritten technique)</span>.
+            <h2>適用 (対象)</h2>
+            <p>リンクが含まれる全てのウェブコンテンツ技術。</p>
+            <p>これは<span><a href="https://waic.jp/docs/WCAG21/Understanding/link-purpose-in-context">達成基準 2.4.4: リンクの目的 (コンテキスト内)</a> (書かれていない達成方法を満たす手段として十分)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to identify the purpose of a link from the link
-               in
-               its data table context. This context is the table cell enclosing the link and the
-               cell's
-               associated table header cells. The data table context provides the purpose for an
-               otherwise
-               unclear link when the table cell is the nearest enclosing block-level ancestor element.
-               It lets a user distinguish this link from other links in the Web page that lead to
-               other
-               destinations and helps the user determine whether to follow the link. Note that simply
-               providing the URI of the destination is not sufficiently descriptive for people with
-               disabilities, especially those with cognitive disabilities.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、リンクとそれを含んでいるデータテーブルの文脈から、リンクの目的を特定することである。文脈とは、そのリンクを含んでいるテーブルセルと、そのセルに関連付けられたテーブルの見出しセルとの関係を指している。データテーブルの文脈によって、リンクを含んでいるセルが、先祖要素であるブロックレベル要素の中で最もそのリンクに近い場合、リンクテキストだけでは不明瞭なリンクに目的が与えられる。それによって、利用者がそのリンクと、そのウェブページ内にある他のリンクとを区別できるようになり、そのリンク先へ移動するかどうかを判断しやすくなる。リンクテキストとして単に目的地の URI を示すだけでは、障害のある利用者、特に認知障害のある利用者にとってはリンク先の説明として不十分であることに注意すべきである。
```

Comment:
宛先→コンテキストに変更しつつ、大幅に手を加えてみました。
おかしなところがないところがないか確認いただければと。
```suggestion
            <p>この達成方法の目的は、リンクとそれを含んでいるデータテーブルのコンテキストから、リンクの目的を特定することである。このコンテキストは、そのリンクを含んでいるテーブルセルと、そのセルに関連付けられたテーブルの見出しセルとの関係を指している。そうでなければデータテーブルのコンテキストは、テーブルセルが最も近いブロックレベルの先祖要素で囲んでいる場合、不明瞭なリンクの目的を提供する。それによって、利用者がこのリンクと他の宛先につながるウェブページの他のリンクを区別できるようになり、そのリンクをたどるかどうかを判断するのに役立つ。単に目的地の URI を示すだけでは、障害のある利用者、特に認知障害のある利用者にとってはリンク先の説明として不十分であることに注意する。
```